### ISSUE TITLE
Add azuredevops_serviceendpoint_maven resource

### ISSUE THREAD

**Issue by drdamour** [microsoft/terraform-provider-azuredevops#857]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

One of the supported service endpoint types in Azure DevOps is `maven`. I would like this to be available as a resource in the azuredevops provider.

### New or Affected Resource(s)

* azuredevops_serviceendpoint_maven

### Potential Terraform Configuration

```hcl
resource "azuredevops_project" "project" {
  name       = "Sample Project"
  visibility         = "private"
  version_control    = "Git"
  work_item_template = "Agile"
}

resource "azuredevops_serviceendpoint_maven" "serviceendpoint" {
  project_id            = azuredevops_project.project.id
  repository_url = "https://example.com"
  repository_id = "ExampleDotCom"
  username = "zero-cool"
  password = "secretTextForPayloadChecksum"
  service_endpoint_name = "Sample Maven"
  description           = "Managed by Terraform"
}
```

### References

* #531 - equivalent request for different endpoint type

**Comment by xuzhang3** [comment]
maven service connection has been added in #617, will available in the next version

**Comment by drdamour** [comment]
wow, a negative 14 day fix speed is pretty fast

**Comment by xuzhang3** [comment]
Available in [v0.8.0](https://registry.terraform.io/providers/microsoft/azuredevops/0.8.0)
